Sources/MongoSwift/MongoDatabase.swift

```diff
@@ -273,15 +273,14 @@ public class MongoDatabase {
     }
 
      /**
-      * Starts a `ChangeStream` on a database. Excludes system collections.
+      * Starts a `ChangeStream` on a database. The server will return an error if this is called on a system collection.
```

Comment:
I think this should go back to the original text about excluding system collections - since a change stream is opened on an entire DB it would be impossible for this method to be used to open a change stream on a system collection.
same applies to other 2 DB watch methods

Reply:
Sorry to ping-pong on this, but I think Kaitlin just meant that database `watch` methods should go back to the old wording. I think for collections it still makes sense to have the error warning.

Reply:
oops, I shouldn't have used find+replace 😅. Will fix.